pos_self_service_weighing_tare/static/src/js/Screens/SelfServiceWeighingScreen/SelfServiceWeighingTareScreen.esm.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module alias=pos_self_service_weighing_tare.SelfServiceWeighingTareScreen **/
+// SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import PosComponent from "point_of_sale.PosComponent";
+import Registries from "point_of_sale.Registries";
+
+class SelfServiceWeighingTareScreen extends PosComponent {}
```

Comment:
Why does this not extend `AbstractSelfServiceWeighingScreen`?

Reply:
done